packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -0,0 +1,549 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import withStyles from '@material-ui/core/styles/withStyles';
+import ButtonBase from '@material-ui/core/ButtonBase';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import clamp from '../utils/clamp';
+
+export const style = theme => {
+  const commonTransitionsOptions = {
+    duration: theme.transitions.duration.short,
+    easing: theme.transitions.easing.easeOut,
+  };
+
+  const commonTransitionsProperty = ['width', 'height', 'box-shadow', 'left', 'top'];
+
+  const commonTransitions = theme.transitions.create(
+    commonTransitionsProperty,
+    commonTransitionsOptions,
+  );
+
+  const colors = {
+    primary: theme.palette.primary.main,
+    secondary: theme.palette.grey[400],
+    focused: theme.palette.grey[500],
+    disabled: theme.palette.grey[400],
+  };
+
+  return {
+    /* Styles for wrapper container */
```

Comment:
I wish we will be able to extra these comments and add it to the API documentation automatically 💯 at some point.